system-int-test/test/cmr/system_int_test/ingest/collection_umm_spec_validation_test.clj

```diff
@@ -137,10 +142,31 @@
       ["AdditionalAttributes"]
       ["Additional Attributes must be unique. This contains duplicates named [bool]."]))
 
+  (testing "Additional Attribute parameter range validation"
```

Comment:
Just added these two tests. For the rest of the changes, I turned on Parinfer and then re-formatted.